vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -149,12 +149,20 @@ public function reconnect() {
 		if ( true === $connection_attempt ) {
 			if ( ! empty( $this->last_heartbeat['cache_site_id'] ) && (int) \Jetpack_Options::get_option( 'id' ) !== (int) $this->last_heartbeat['cache_site_id'] ) {
 				$this->send_alert( 'Alert: Jetpack was automatically reconnected, but the connection may have changed cache sites. Needs manual inspection.' );
-
 				return;
 			}
 
 			$this->send_alert( 'Jetpack was successfully (re)connected!' );
 
+			$vaultpress_connection_attempt = Connection_Pilot\Controls::connect_vaultpress();
+
+			if ( is_wp_error( $vaultpress_connection_attempt ) ) {
+				$this->send_alert( 'Alert: Could not connect VaultPress automatically.' );
```

Comment:
Can pass the wp_error as the second arg here, then the logging will give some more details.

Since it's a VP connection error, can make this line more generic: https://github.com/Automattic/vip-go-mu-plugins/blob/af245ce05ced3135c287f92bd992eb90ce28faf7/vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php#L251